inc/Debug/PropertyDeprecationHelper.php

```diff
@@ -108,7 +108,7 @@ private function deprecationHelperGetPropertyOwner($property)
             // The class name is not necessarily correct here but getting the correct class
             // name would be expensive, this will work most of the time and getting it
             // wrong is not a big deal.
-            return __CLASS__;
+            return self::class;
```

Comment:
I don't see the reason for this (type of) change. AFAICT the result is identical?

Reply:
https://stackoverflow.com/a/75650590 says it's somehow better without explicitly explaining why :man_shrugging: 

I guess one seems more like a preprocessor replacement while the other is a proper language construct? I have no idea, but I'm fine with going with the "more modern" choice.